templates/miq-template-ext-db.yaml

```diff
@@ -265,7 +265,7 @@ objects:
   spec:
     ports:
       -
-        name: "postgresql"
+        name: "postgres"
```

Comment:
These are the official templates and naming from OpenShift Origin, don't recall a postgres reference for the service before, just saying... 

https://github.com/openshift/origin/blob/master/examples/db-templates/postgresql-persistent-template.json